src/plugins/query_enhancements/public/plugin.tsx

```diff
@@ -69,6 +69,45 @@ export class QueryEnhancementsPlugin
       editor: createEditor(SingleLineInput, null, pplControls, DefaultInput),
       editorSupportedAppNames: ['discover'],
       supportedAppNames: ['discover', 'data-explorer'],
+      sampleQueries: [
+        {
+          title: i18n.translate('queryEnhancements.sampleQuery.titleContainsWind', {
+            defaultMessage: 'The title field contains the word wind.',
```

Comment:
Should we be quoting the field names? ... or wrapping them in `<code />`

Reply:
https://github.com/opensearch-project/OpenSearch-Dashboards/issues/8766